sunpy/visualization/wcsaxes_compat.py

```diff
@@ -29,10 +26,7 @@ def is_wcsaxes(axes):
     `bool`
         Result of the test.
     """
-    if not _FORCE_NO_WCSAXES:
-        return isinstance(axes, wcsaxes.WCSAxes)
-    else:
-        return False
+    return isinstance(axes, wcsaxes.WCSAxes)
```

Comment:
Not sure we need this method anymore but we can punt down the road.

Reply:
Yeah, might as well wait till 3.0 since a deprecation needs to be in one LTS anyway.